.github/workflows/nightly.yml

```diff
@@ -0,0 +1,44 @@
+name: Nightly Tests
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '00 19 * * *'
+
+jobs:
+  stress-test:
+    runs-on: self-hosted
```

Comment:
This isn't a good name for the runner.
How do I distinguish this from lets say a windows or mac runner?

Reply:
I can't control that name. You have to use that label, and you can use additional ones, but they're all linux.

Reply:
Right, you'd need to use an array of labels.
For the M1 Mac, we have: [self-hosted, MacOS, arm64]